roles/openshift_control_plane/tasks/upgrade.yml

```diff
@@ -56,8 +56,7 @@
   yedit:
     src: "{{ openshift.common.config_base }}/master/master-config.yaml"
     key: 'imageConfig.format'
-    value: "{{ oreg_url | default(oreg_url_master) }}"
-  when: oreg_url is defined or oreg_url_master is defined
+    value: "{{ openshift_imageconfig_format }}"
```

Comment:
We need this to ensure registry.access.redhat.com is migrated to registry.redhat.io;  We can't set oreg_url by force for various reasons.